packages/suite/src/support/messages.ts

```diff
@@ -3158,11 +3158,40 @@ export default defineMessages({
         id: 'TR_TOR_TITLE',
         defaultMessage: 'Tor',
     },
+    TR_TOR_ENABLE: {
+        id: 'TR_TOR_ENABLE',
+        defaultMessage: 'Enable Tor',
+    },
+    TR_TOR_DISABLE: {
+        id: 'TR_TOR_DISABLE',
+        defaultMessage: 'Disable Tor',
+    },
+    TR_TOR_ENABLE_AND_CONFIRM: {
+        id: 'TR_TOR_ENABLE_AND_CONFIRM',
+        defaultMessage: 'Enable Tor and confirm',
+    },
     TR_TOR_DESCRIPTION: {
         id: 'TR_TOR_DESCRIPTION',
         defaultMessage:
             "Enabling this will route all of Suite's traffic through the Tor network.{lineBreak}All requests to Trezor infrastructure will be pointed to our Tor services, increasing your privacy and security.",
     },
+    TR_TOR_REMOVE_ONION_AND_DISABLE: {
+        id: 'TR_TOR_REMOVE_ONION_AND_DISABLE',
+        defaultMessage: 'Remove all addresses and disable Tor',
```

Comment:
This has nothing to do with this text. I see some product and design issues here.

<img width="735" alt="Screenshot 2022-03-01 at 10 11 28" src="https://user-images.githubusercontent.com/33235762/156139638-2d484a7b-43fd-4c94-b0cd-43a856856b89.png">

Design: 
- small padding around backends (see design image)
- `Go to settings` button looks weird 
- I like the texts in design more, why was it changed?

Product:
- I miss an option to disable tor without removing my onion addresses. Is there a case, that I need to disable tor to do something?
- What about having an option to provide both normal and tor addresses?

<img width="718" alt="Screenshot 2022-03-01 at 10 18 49" src="https://user-images.githubusercontent.com/33235762/156140812-a9c68af9-e8c4-4a9c-8cef-815d8dabcb5a.png">
In this case, it can just fall back to second address